packages/dd-trace/test/noop.spec.js

```diff
@@ -41,6 +41,14 @@ describe('NoopTracer', () => {
     })
   })
 
+  describe('flush', () => {
+    it('should return the function', () => {
+      const fn = () => {}
+
+      expect(tracer.flush(fn)).to.equal(fn)
```

Comment:
This and other similar tests should test instead that the function is called, not that it is returned.